coalib/coala_main.py

```diff
@@ -91,7 +91,7 @@ def run_coala(console_printer=None,
                                     actions. If you set this to `False`,
                                     `force_show_patch` will be ignored.
     :param force_show_patch:        If set to True, a patch will be always
-                                    shown. (Using ApplyPatchAction.)
+                                    shown (Using ApplyPatchAction).
```

Comment:
The changes you made aren't relating to the actual issue.
Please carefully read the issue description